governance-and-automation/security-and-compliance-checks.md

```diff
@@ -0,0 +1,139 @@
+---
+description: This article provides information on the Security and Compliance checks feature in SysKit Point.
+---
+
+# Security and Compliance Checks: Early Access 
+
+The new **Security and Compliance Checks** visible directly from your SysKit Point home screen pinpoint the **most important vulnerabilities and misalignments** with Microsoft 365 best practices. These checks should keep you aware and help you **react quickly to keep your environment secure and under control**.
```

Comment:
nešto mi malo čudno zvuči u ovoj rečenici, jel fali možda zarez poslije **security and compliance checks**? ne znam kako da objasnim al mi rečenica malo zvuči ko u jednom dahu, ja bi možda napisao ovako:

The new **Security and Compliance Checks**, visible directly from your SysKit Point home screen helps you pinpoint the most important vulnerabilities and misalignments with Microsoft 365 best practices. These checks should keep you aware and help you **react quickly to keep your environment secure and under control**.

ali možda sam i u krivu, no svakako bi napravio nešto da dam rečenici prostora za "disanje" so to speak.